restmanager/fridges/urls.py

```diff
@@ -2,6 +2,7 @@
 from .api import FridgeViewSet
 
 router = routers.DefaultRouter()
+# registers the api/fridges endpoint and maps it to use FridgeViewSet
 router.register('api/fridges', FridgeViewSet, 'fridges')
```

Comment:
Maybe you want to have versioned API: https://restfulapi.net/versioning/

Reply:
Osoittautui ennakoitua haastavammaksi, katsotaan saadaanko tehtyä.